app/src/main/res/layout/auth_ui_layout.xml

```diff
@@ -146,6 +146,28 @@
                 android:layout_height="wrap_content"
                 android:layout_marginBottom="8dp"
                 android:layout_marginTop="16dp"
+                android:text="@string/privacy_policy_header"/>
+
+            <RadioGroup
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <RadioButton
+                    android:id="@+id/google_privacy"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:checked="true"
+                    android:text="@string/google_privacy_label"/>
```

Comment:
I was looking for some kind of Fireabse privacy policy, but I could only find terms of service. That's a good idea.